### PR TITLE
add task

### PR DESCRIPTION
**Pull request by AnnetaKozyreva**

### CODE REVIEW

File: src/UniversityGroup.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    UniversityGroup(String gName, int sYear, ArrayList<String> sList) {
+        groupName = gName;
+        startYear = sYear;
+        endYear = startYear + 5;
+        studentList = sList;
+
+    }
+
+    UniversityGroup() {
```
Comment (ltana) [inline src/UniversityGroup.java]: why do you need this?

File: src/UniversityGroup.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        System.out.println("===========================================================");
+        System.out.println("Group name: " + getGroupName());
+        System.out.println("Start year: " + getStartYear());
+        System.out.println("End year: " + getEndYear());
+        System.out.println("Students: " + String.join(", ", getStudentList()));
+    }
+
+
+        public static void main(String[] args) {
+            ArrayList<String> studentArray = new ArrayList<String>(List.of("Student11", "Student12", "Student13", "Student14", "Student15"));
```
Comment (ltana) [inline src/UniversityGroup.java]: try to add students using methods for collection from last lection

**Review by ltana — CHANGES_REQUESTED** [review]

File: src/UniversityGroup.java (hunk 2 above)
Comment (AnnetaKozyreva) [reply src/UniversityGroup.java]: I fixed the notes.

**Review by ltana — APPROVED** [review]
can be merged

**Pull request merged**